exporter/top_collector_test.go

```diff
@@ -42,10 +41,10 @@ func TestTopCollector(t *testing.T) {
 	// 1. The result is huge
 	// 2. We need to check against know values. Don't use metrics that return counters like uptime
 	//    or counters like the number of transactions because they won't return a known value to compare
-	filter := []string{
-		"mongodb_top_update_count",
-	}
-	filter = nil
+	//filter := []string{
```

Comment:
Leftover?

Reply:
I have no idea what happened there, so I decided to leave it as a comment to make linters not complain about that.